src/components/shared/ExternalLinkMenuItem.tsx

```diff
@@ -0,0 +1,35 @@
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import { Link, MenuItem } from '@mui/material';
+import PropTypes from 'prop-types';
+
+ExternalLinkMenuItem.propTypes = {
+    children: PropTypes.any.isRequired,
+    link: PropTypes.string.isRequired,
+};
+
+function ExternalLinkMenuItem(
+    props: PropTypes.InferProps<typeof ExternalLinkMenuItem.propTypes>
+) {
+    return (
+        <MenuItem
+            component={Link}
+            href={props.link}
+            target="_blank"
+            rel="noopener"
+            tabIndex={0}
+            sx={{
+                px: 3,
+                py: 1.5,
+                fontSize: 14,
+                fontWeight: 700,
+                color: '#3c5584',
```

Comment:
A TODO is missing from this line to indicate that the hex color code should be replaced with a theme asset. If you would like that comment added before the code is merged, let me know.